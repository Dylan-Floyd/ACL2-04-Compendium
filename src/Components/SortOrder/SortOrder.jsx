import React from 'react'
import Select from '../Select/Select.jsx'

export default function SortOrder({
  selectedOrder,
  handleChange
}) {
  return (
    <Select label='Sort Order: ' optionsList={['↑', '↓']} selectedOption={selectedOrder} handleChange={handleChange} />
  )
}
