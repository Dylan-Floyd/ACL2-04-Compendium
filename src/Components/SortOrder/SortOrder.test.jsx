import { render } from '@testing-library/react'
import SortOrder from './SortOrder.jsx'

it('Should render the SortOrder component', () => {
  const { container } = render(<SortOrder />)
  expect(container).toMatchSnapshot()
})