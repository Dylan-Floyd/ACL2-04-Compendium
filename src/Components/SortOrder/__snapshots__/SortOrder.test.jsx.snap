// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render the SortOrder component 1`] = `
<div>
  <label>
    Sort Order: 
    <select>
      <option
        value="↑"
      >
        ↑
      </option>
      <option
        value="↓"
      >
        ↓
      </option>
    </select>
  </label>
</div>
`;
